client/my-sites/domains/domain-management/components/designated-agent-notice/index.jsx

```diff
@@ -1,7 +1,8 @@
 import { Gridicon } from '@automattic/components';
 import { localizeUrl } from '@automattic/i18n-utils';
-import { DESIGNATED_AGENT, DOMAIN_REGISTRATION_AGREEMENTS } from '@automattic/urls';
```

Comment:
The `DESIGNATED_AGENT` constant, is no longer used. Can we remove it?

Reply:
Removed!